exonum/src/blockchain/tests.rs

```diff
@@ -298,6 +391,18 @@ mod memorydb_tests {
         )
     }
 
+    fn create_blockchain_with_service(_: &Path, service: Box<Service>) -> Blockchain {
+        let service_keypair = gen_keypair();
+        let api_channel = mpsc::channel(1);
+        Blockchain::new(
+            Box::new(MemoryDB::new()),
```

Comment:
Why doesn't this use `create_database`? +The comment about macros is applicable here as well (albeit a bit off-topic). You could look at #380 for a reference how I envision tests with macros.

Reply:
Because `create_database` required an additional variable `Path` 😄